upgrade/kind.go

```diff
@@ -192,28 +192,19 @@ var getRepoKind = stepv2.Func11E("Get Repo Kind", func(ctx context.Context, repo
 	}
 
 	out.UpstreamProviderOrg = stepv2.Func11("Get UpstreamOrg", func(ctx context.Context, path string) string {
```

Comment:
It would be great to separate out `stepv2.Func11("Get UpstreamOrg", ...` into a standalone function (`var getUpstreamOrg = stepv2.Func11("Get UpstreamOrg", ...`) and put it under test.

Also, `path` is no longer used, so we can remove it.

Reply:
Done